parser/src/test/java/com/facebook/coresql/parser/sqllogictest/java/SqlLogicTestResult.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.coresql.parser.sqllogictest.java;
+
+import com.facebook.coresql.parser.sqllogictest.java.SqlLogicTest.AcceptedRdbms;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class SqlLogicTestResult
+{
+    private final List<ErroneousSqlStatement> statementsThatLedToError;
```

Comment:
You can just name the variable "errorneousSqlStatements" or "errorneousStatements"

Reply:
Done